tests/build_fake_data.py

```diff
@@ -76,7 +75,7 @@
                            timestamps=spatial_timestamps,
                            resolution=0.1,
                            comments="This data was generated with numpy, using 1234 as the seed",
```

Comment:
FWIW, this comment is incorrect. 1234 is used as the seed before the fake electrical series gets generated. This data uses the next seed.

Reply:
@ajtritt Should we address the random number issues that @neuromusic and @nicain raised in another PR?

Reply:
There is no issue. The random number generator is already seeded with a constant seed.

Reply:
The seed isn't reset between 

https://github.com/NeurodataWithoutBorders/pynwb/blob/43dfb293b79a4303d4ea1629706132c19b95c0da/tests/build_fake_data.py#L54

and

https://github.com/NeurodataWithoutBorders/pynwb/blob/43dfb293b79a4303d4ea1629706132c19b95c0da/tests/build_fake_data.py#L57

so the comment in 

https://github.com/NeurodataWithoutBorders/pynwb/blob/43dfb293b79a4303d4ea1629706132c19b95c0da/tests/build_fake_data.py#L77

is false.

If the parameters used to generate the size of the first array change, this will change the second array.

![image](https://user-images.githubusercontent.com/1245615/32070080-f685da20-ba3f-11e7-8b93-ff71b092ee9d.png)